packages/OverflowMenu/src/OverflowMenu.js

```diff
@@ -35,6 +35,9 @@ const propTypes = {
 
   /** The z-index for the popover / confirmation */
   zIndex: PropTypes.number,
+
+  /** The maximum height of the OverflowMenu content */
+  maxHeight: PropTypes.number,
```

Comment:
Some of our components allow `number` or `string` for these kinds of values (I know I recently updated the `<Textarea>` to allow this).  That makes it a little more flexible, for instance, I could set the `maxHeight` to `45vh` – which actually might make a nice default value instead of `null`.  

Probably worth discussing the default behaviour with @sunminsarahlee

Reply:
If we go with vh, the height will be different depending on the screen size so I’d rather prefer fix value so it will be consistent for all users. So I will propose 300px which can fit 8 items/lines in the dropdown.

Reply:
for the default value we could use `vh`